src/sentry/static/sentry/app/data/forms/accountNotificationSettings.jsx

```diff
@@ -1,88 +1,82 @@
+import {t, tct} from '../../locale';
+
 // Export route to make these forms searchable by label/help
 export const route = '/settings/account/notifications/';
 
 const formGroups = [
   {
-    title: 'Alerts',
+    title: t('Alerts'),
     fields: [
       {
         name: 'subscribeByDefault',
         type: 'boolean',
-        label: 'Send Me Project Alerts',
-        help: 'Alerts are defined in [Project] » Project Settings » Alerts » Rules.',
+        label: t('Send Me Project Alerts'),
+        // TODO(billy): Make this a real link
+        help: tct('Alerts are defined in [locationPath]', {
```

Comment:
It's returning a React element, field from config complains because it wants a string or function